.pipelines/templates/.builder-release-template.yaml

```diff
@@ -49,7 +49,6 @@ steps:
         **/kata-containers-cc.img**
         **/kata-containers.img**
         **/kata-containers-initrd-base.img**
-        **/reference-info-base64**
```

Comment:
This task should no longer be built anyway since we have frozen the kata-cc image and set the default build to false for that VHD.

Reply:
I'm ok with either removing it or leaving it as is, but this change should not have any effect.

Reply:
So this is not being referenced right? We should clean it up then.